python/cugraph/structure/symmetrize.py

```diff
@@ -54,26 +54,30 @@ def symmetrize_df(df, src_name, dst_name):
     >>> sym_ddf = cugraph.symmetrize_ddf(ddf, "src", "dst", "weight")
     >>> Comms.destroy()
     """
-    gdf = cudf.DataFrame()
-
     #
     #  Now append the columns.  We add sources to the end of destinations,
     #  and destinations to the end of sources.  Otherwise we append a
     #  column onto itself.
     #
-    for idx, name in enumerate(df.columns):
-        if name == src_name:
-            gdf[src_name] = df[src_name].append(
-                df[dst_name], ignore_index=True
-            )
-        elif name == dst_name:
-            gdf[dst_name] = df[dst_name].append(
-                df[src_name], ignore_index=True
-            )
-        else:
-            gdf[name] = df[name].append(df[name], ignore_index=True)
-
-    return gdf.groupby(by=[src_name, dst_name], as_index=False).min()
+    if symmetrize:
```

Comment:
We should talk about, and document, what symmetrize means for Multigraphs.  
 Assume the there are edges: A-B, A-B, A-C, B-A.     
Is the B-A edge unique and should be  symmetrize, or is it already a reverse of one of the A-B edges.  
Currently there are two edges from A to B, and one from B to A.  Should symmetrize produce three edges between the pair or two?

Reply:
As per our MultiGraph discussion we concluded that in cases like A-B, A-B, A-C, B-A, there should be no assumption made and every edge should be treated as unique.